src/ch/blinkenlights/android/medialibrary/MediaLibrary.java

```diff
@@ -123,6 +124,7 @@ private static MediaLibraryBackend getBackend(Context context) {
 	 * @param context the context to use
 	 * @return MediaLibrary.Preferences
 	 */
+	@TargetApi(Build.VERSION_CODES.KITKAT)
```

Comment:
Shouldn't we fix this? We have minSdkVersion == 14 and this function can be called on < Kitkat versions, which will result in a crash in try-with-resources clause.